pom.xml

```diff
@@ -90,6 +90,8 @@
         <zkclient.version>0.11</zkclient.version>
         <zookeeper.version>3.5.9</zookeeper.version>
         <bouncycastle.version>1.68</bouncycastle.version>
+        <jmx_prometheus_javaagent.version>0.14.0</jmx_prometheus_javaagent.version>
+        <jolokia-jvm.version>1.6.2</jolokia-jvm.version>
```

Comment:
These versions were selected based on what we currently have [here](https://github.com/confluentinc/confluent-platform-images/blob/release-6.1.x/components/common/Dockerfile#L24-L31).

@xnulinu Please let me know if we should use different versions.

Reply:
@jhlee8 yes